test/elixir_search_extractor/search_keyword/queries/keyword_query_test.ex

```diff
@@ -0,0 +1,15 @@
+defmodule ElixirSearchExtractor.SearchKeyword.Queries.KeywordQueryTest do
+  use ExUnit.Case, async: true
+
+  alias ElixirSearchExtractor.SearchKeyword.Queries.KeywordQuery
+
+  describe "filter_keywords/2" do
+    test "it returns unmodified query if the second argument is nil" do
```

Comment:
```suggestion
    test "returns unmodified query if the second argument is nil" do
```

Reply:
@rosle removed in [5845596](https://github.com/rafayet-monon/elixir_search_extractor/pull/44/commits/5845596a9b46caf352b7ff1abd2c886664528970)